content-accelerator/latest/install/install-guide.md

```diff
@@ -308,22 +312,47 @@ In this section we ensure that all components of the Content Accelerator are ins
    * If using Alfresco Content Services 7.2.x, use the `tsgrp-opencontent-3.5-for-acs7.2.amp`.
    * If using Alfresco Content Services 7.3.x, use the `tsgrp-opencontent-3.5-for-acs7.3.amp`.
 
-3. (Pnp ONLY) This step is only required if installing the Policy and Procedure Content Accelerator solution:
+3. (Pnp and HR ONLY) This step is only required if using the Policy and Procedure Content Accelerator or HR Content Accelerator solution:
 
-   Navigate to the `ALFRESCO_HOME/amps` directory and copy the following amps there:
-   * `tsgrp-alfresco-chain-versioning.amp`
-   * `pnp-platform-3.5.amp`
+   Copy the `WizardAdmin.war` file into the `ALFRESCO_HOME/tomcat/webapps` directory.
 
-   These amps can be found in the alfresco-content-accelerator-policy-and-procedure-accelerator distribution zip under `Alfresco Artifacts` folder.
+   You'll find this WAR file in the `Web Applications` folder of the `alfresco-content-accelerator-policy-and-procedure-accelerator` zip or `alfresco-content-accelerator-sehr-accelerator` zip.
```

Comment:
Need to revert this section & move it to step 3 in `install-guide/#install-webapps` section .. _to both subsections_? 🤔

Reply:
Step 3 moved by commit [f081bee](https://github.com/Alfresco/docs-alfresco/pull/1141/commits/f081bee19ecff9b89e69d7f1142c7e199a221588)

Reply:
Both Tomcat subsection updated - see latest commit [651cb0b](https://github.com/Alfresco/docs-alfresco/pull/1141/commits/651cb0b620807d13b39bf8090b9420b96dd5ce35)